product_code/lagom/course_service/api/src/main/scala/de/upb/cs/uc4/course/model/Course.scala

```diff
@@ -53,39 +39,42 @@ case class Course(
    */
   def validateCourseSyntax: Seq[SimpleError] = {
 
-    val course = this
-    val nameRegex = """[\s\S]*""".r // Allowed characters for coursename "[a-zA-Z0-9\\s]+".r
-    val descriptionRegex = """[\s\S]*""".r // Allowed characters  for description
-    val dateRegex = """(\d\d\d\d)-(\d\d)-(\d\d)""".r
+    val nameRegex = """[\s\S]{1,100}""".r // Allowed characters for coursename "[a-zA-Z0-9\\s]+".r
```

Comment:
Correct me if I am wrong, but the comment seems off
```suggestion
    val nameRegex = """[\s\S]{1,100}""".r // Allowed characters for coursename: 1-100 of everything
```